index.html

```diff
@@ -156,6 +156,58 @@ <h3>If the checks will be performed by a different organisation than the one tha
               be able to make the deed effective.
             </p>
           </p>
+        </div>
+        <div class="schemas">
+          <a name="auth-headers"><h2>User Authentication</h2></a>
+          <p>
+            To be able to use this service, you will need to include an authorization header in your
+            requests to our service.  
+          </p>
+          <p>
+            The correct format for the Authorization header is to add your username and password seperated by a colon.
+            As an example you could have 'rodneysteel' as the username and 'password' as the password. Which would look like:
+<pre>
+  <code>
+  rodneysteel:password
+  </code>
+</pre>
+            Then Base64 encode this string, would which look like:
+<pre>
+  <code>
+  cm9kbmV5c3RlZWw6cGFzc3dvcmQ=
+  </code>
+</pre>
+            Finally, you will need to add this Base64 encoded username and password to your Authorization header for the request.
+<pre>
+  <code>
+  Key: Authorization
+  Value: Basic cm9kbmV5c3RlZWw6cGFzc3dvcmQ=
+  </code>
+</pre>
+          </p>
+
+          <p>
+            If the request is successfull, the expected payload for the enpoint will be returned. If however there is an issue with your username and password, the following responses may be returned:
+          </p>
+401 UNAUTHORIZED
+<pre>
+  <code>
+  { "Message": "Invalid credentials." }
+  </code>
+</pre>
+
+          <p>
+            If no header is sent with the request:
```

Comment:
In the future,  Authorization headers will be mandatory.  This will be communicated to conveyancers nearer the time.